docs/sources/reference/commandline/cli.md

```diff
@@ -2336,7 +2336,7 @@ Running `docker stats` on multiple containers
     $ docker stats redis1 redis2
     CONTAINER           CPU %               MEM USAGE/LIMIT     MEM %               NET I/O
     redis1              0.07%               796 KiB/64 MiB      1.21%               788 B/648 B
```

Comment:
@wonderflow not sure, better look if there's an existing pull-request/issue for that first. If you can't find one, feel free to create a pull-request, then it can be discussed, just to be sure the maintainers agree :)